system/windows.js

```diff
@@ -0,0 +1,12 @@
+serenade.global().command("curly brackets in",async api =>{
+    await api.typeText("{}");
+    await api.pressKey("left")
+});
+serenade.global().command("curly brackets out",async api =>{
+    //await api.focus("terminal");
```

Comment:
There's no need for this comment here which I believe is leftover from copying an example?